django/db/migrations/state.py

```diff
@@ -29,6 +29,29 @@ def _get_app_label_and_model_name(model, app_label=''):
         return model._meta.app_label, model._meta.model_name
 
 
+def _get_related_models(m):
+    """
+    Returns all model that have a direct relationship to the given model.
```

Comment:
new style is "Return all models...". I think the "private API" bit is implied by the underscore prefix, but doesn't hurt to be explicit if you like.